packages/marketplace/src/components/pages/developer-submit-app/form-schema/validation-schema.ts

```diff
@@ -0,0 +1,17 @@
+import * as Yup from 'yup'
+import formFields from './form-fields'
+import errorMessages from '@/constants/error-messages'
+import { letterNumberSpaceRegex } from '@reapit/elements'
+
+const { FIELD_REQUIRED } = errorMessages
+
+const { name } = formFields
+
+const submitAppValidationSchema = Yup.object().shape({
+  [name.name]: Yup.string()
+    .trim()
```

Comment:
should be `.trim().required(FIELD_REQUIRED).matches(letterNumberSpaceRegex, name.errorMessage)` so that when users enter spaces, we will show the error message from `FIELD_REQUIRED` instead of `name.errorMessage`